src/types.ts

```diff
@@ -366,10 +367,12 @@ export type ResourceGuesserProps = Omit<
 >;
 
 type CreateSimpleFormProps = Omit<
-  CreateProps & Omit<SimpleFormProps, 'component'>,
+  CreateProps & SimpleFormProps & TabbedFormProps,
```

Comment:
Why the `component` prop is not omitted anymore?

Reply:
Is it not redundant here? 

Intersection with `Patial<PickRename>` to replace "component" with "formComponent" should do the thing. I just used as a pattern simplified syntax from `EditSimpleFormProps` to unify it both statements :
https://github.com/api-platform/admin/blob/3762ad0a8ffb300554df77b6804ce751e720c7e5/src/types.ts#L388